src/components/Card.js

```diff
@@ -34,19 +48,36 @@ export default class Card {
       this._handleImageClick(this._name, this._link);
     });
     this._likeButton.addEventListener("click", () => {
-      this._handleLikeIcon();
+      if (this._isLiked) {
+        this._removeLike(this);
+      } else {
+        this._likeCard(this);
+      }
     });
     this._trashButton.addEventListener("click", () => {
-      this._handleDeleteButton();
+      this._handleDeleteClick(this);
     });
   }
 
-  _handleDeleteButton() {
+  handleDeleteCard() {
```

Comment:
Needs correction: please rename this method to something like `deleteCard`, since it's not an event listener anymore